### ISSUE TITLE
Keyboard user must be able to close sidenav and not lose focus

### ISSUE THREAD

**Issue by marcysutton** [angular/material#728]
After a sidenav opens, the escape key closes the element it but focus is lost, requiring the user to tab back to where they were. Focus should be sent back to the triggering item. (Md-dialog does this correctly.)
